pgmpy/inference/ExactInference.py

```diff
@@ -248,134 +248,104 @@ def query(
         joint=True,
         show_progress=True,
     ):
-        """
-        Parameters
-        ----------
-        variables: list
-            list of variables for which you want to compute the probability
+    """
+    Parameters
+    ----------
+    variables: list
+        list of variables for which you want to compute the probability
 
-        evidence: dict
-            a dict key, value pair as {var: state_of_var_observed}
-            None if no evidence
+    evidence: dict
+        a dict key, value pair as {var: state_of_var_observed}
+        None if no evidence
 
-        virtual_evidence: list (default:None)
-            A list of pgmpy.factors.discrete.TabularCPD representing the virtual
-            evidences.
+    virtual_evidence: list (default:None)
+        A list of pgmpy.factors.discrete.TabularCPD representing the virtual
+        evidences.
 
-        elimination_order: str or list (default='greedy')
-            Order in which to eliminate the variables in the algorithm. If list is provided,
-            should contain all variables in the model except the ones in `variables`. str options
-            are: `greedy`, `WeightedMinFill`, `MinNeighbors`, `MinWeight`, `MinFill`. Please
-            refer https://pgmpy.org/exact_infer/ve.html#module-pgmpy.inference.EliminationOrder
-            for details.
+    elimination_order: str or list (default='greedy')
+        Order in which to eliminate the variables in the algorithm.
 
-        joint: boolean (default: True)
-            If True, returns a Joint Distribution over `variables`.
-            If False, returns a dict of distributions over each of the `variables`.
+    joint: boolean (default: True)
+        If True, returns a Joint Distribution over `variables`.
```

Comment:
Could you change this to something like "If True, returns a joint posterior distribution over `variables`. Otherwise, returns marginal posterior distributions over each variable"